pipeline/utilities/variantsBySource.py

```diff
@@ -24,7 +24,7 @@ def main():
                               "ExAC": 0,
                               "BIC": 0,
                               "ESP": 0,
-                              "Findlay_BRCA1_Ring_Function_Scores": 0,
```

Comment:
maybe include both for historical reasons?

Reply:
Good point.  Something to consider is the impact of these changes on variant history.  For the variants that are covered by the Findlay assay, the same assay results should be in the variant record from the ENIGMA assays moving forward, but looking back at legacy data is another matter.  Maybe including both is cleaner for managing the legacy data.

Reply:
yea, we'll want to review this in beta. i anticipate issues viewing legacy variants and we'll need to decide what's worth supporting